graylog-storage-elasticsearch7/src/main/java/org/graylog/storage/elasticsearch7/views/searchtypes/pivot/buckets/ESValuesHandler.java

```diff
@@ -105,7 +105,9 @@ private TermsAggregationBuilder createScriptedTerms(List<? extends BucketSpec> b
 
     private Script scriptForPivots(Collection<? extends BucketSpec> pivots) {
         final String scriptSource = Joiner.on(KEY_SEPARATOR_PHRASE).join(pivots.stream()
-                .map(bucket -> "String.valueOf(doc['" + bucket.field() + "'].size() == 0 ? \"N/A\" : doc['" + bucket.field() + "'].value)")
+                .map(bucket -> """
+                        String.valueOf((doc.containsKey('%s') && doc['%s'].size() > 0) ? doc['%s'].value : "%s")
```

Comment:
What about "%1$s" (so called "Argument Index") format string and using "bucket.field()" only once?

Reply:
Oh great, I was looking for something like this! :pray: